src/file_upload.rs

```diff
@@ -172,8 +218,9 @@ pub fn upload_file(
         }
     };
 
-    // If the target path is under the app root directory, save the file.
+    // if no_symlinks, make sure the target dir is under the app root directory
```

Comment:
```suggestion
    // if 'conf.no_symlinks', make sure the canonical target path is under the app root directory
```
Is this any good?

Reply:
Let's use backticks instead of singlequotes and also perhaps mention the upload aspect of this whole thing (it was in the original comment, too). After all, that's what this code branch is for. :)